attalos/dataset/visgenome-densecap/visgenome2attalos.py

```diff
@@ -0,0 +1,201 @@
+import sys
+sys.path.append('/home/kni/local-kni/attalos')
+from attalos.dataset.dataset import Dataset
+# from attalos.dataset.transformers.onehot import OneHot
+import numpy as np
+
+
+datadir='/data/fs4/teams/attalos/features/'
+imdata=datadir+'image/visualgenome_train_20160816_inception.hdf5'
+txdata=datadir+'text/visualgenome_train_20160816_text.json.gz'
+dirTr = '/data/fs4/datasets/vg_unzipped/'
+
+alldata = Dataset(imdata,txdata)
+
+for key in alldata.text_feats:
+    words = alldata.text_feats[key]
```

Comment:
Could you add a comment about what this is doing (That is, taking the last word)?